src/SupervisionSuccess/components/Outcomes/Outcomes.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./Outcomes.css";
+
+const Outcomes = ({ prisonPopulationDiff, savings }) => {
+  const [prisonPopulationDiffText, prisonPopulationDiffIconClass] = (() => {
+    if (prisonPopulationDiff < 0) return ["Fewer people in prison", "outcomes_stat-icon-down"];
+    if (prisonPopulationDiff > 0) return ["More people in prison", "outcomes_stat-icon-up"];
+    return [null, ""];
+  })();
+
+  const [savingsText, savingsIconClass] = (() => {
+    if (savings > 0) return ["Reduced costs", "outcomes_stat-icon-down"];
+    if (savings < 0) return ["Increased costs", "outcomes_stat-icon-up"];
+    return [null, ""];
```

Comment:
What does this render as if the value is actually 0? Is it just completely blank with no text and no icon? (Same question for prison population diff just above.)

I see the PRD didn't specify this behavior so I'll ping Andrew to check in on that case on the PRD and we can circle back here.

Reply:
Yes, now no icon and blank text.